docs/src/main/sphinx/release/release-383.md

```diff
@@ -0,0 +1,113 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregation. ({issue}`12336`)
+* Add IAM Role support for exchange spooling on S3. ({issue}`12444`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
+
+## Atop connector
+
+## BigQuery connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Support storing a table comment when creating new tables. ({issue}`12452`)
```

Comment:
Make that consistent wth the above ones about using CREATE TABLE  ... maybe by rewording the above entries